ERCS/erc-Upgradeable_Clone.md

```diff
@@ -0,0 +1,199 @@
+---
+erc:
```

Comment:
```suggestion
erc: 7546
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.